app/resources/js/dev_visual_optimizer_polyfill.js

```diff
@@ -0,0 +1 @@
+(function () { if (window._vwo_code && window._vwo_code.finish) { window._vwo_code.finish(); })());
```

Comment:
This seems to have a syntax error

Maybe:
```suggestion
(function () { if (window._vwo_code && window._vwo_code.finish) { window._vwo_code.finish(); } })();
```